icu_benchmarks/data/split_process_data.py

```
@@ -39,7 +42,7 @@
     complete_train: bool = False,
     runmode: RunMode = RunMode.classification,
     label: str = None,
-    required_var_types=["GROUP", "SEQUENCE", "LABEL"],
+    required_var_types=["GROUP", "LABEL"],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)